tests/utils/database.test.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable mocha/no-hooks-for-single-case */
+/* eslint-disable func-names */
```

Comment:
I like these disabled as well. Should we move them to a `.eslintrc.js` in the tests directory? If you have trouble getting that working with typescript, then feel free to open a new Project ticket for us to update the linter.

Reply:
Added a new eslintrc file.